tests/dialects/test_bigquery.py

```diff
@@ -170,7 +170,7 @@ def test_bigquery(self):
                 "bigquery": "CURRENT_TIMESTAMP()",
                 "duckdb": "CURRENT_TIMESTAMP()",
                 "postgres": "CURRENT_TIMESTAMP",
-                "presto": "CURRENT_TIMESTAMP()",
+                "presto": "CURRENT_TIMESTAMP",
```

Comment:
Checked both docs & ran it locally -- `current_timestamp` is supposed to be a no-parentheses function.